packages/geoprocessing/src/toolbox/geoblaze/rasterStats.ts

```diff
@@ -53,9 +56,9 @@ export const rasterStats = async (
   // If area is included in stats list, then also include valid stat which is needed to calculate area later
   if (stats.includes("area")) statsToCalculate.push("valid");
   // If categorical raster, only calculate valid stat
-  if (category) {
-    statsToCalculate = ["valid"];
-    statsToPublish = ["valid"];
+  if (categorical) {
+    statsToCalculate = ["histogram"];
+    statsToPublish = ["histogram"];
```

Comment:
I would suggest using the "count" stat.  Thoughts?  See Concepts.md for list of stats supported.

Reply:
In my mind `histogram` describes something bigger than the individual count/frequency stats it calculates.  Like together the individual stats bundled together can be used to make a frequency chart.

Reply:
Yeah, I've been going back and forth on this, because at the end it does turn into the "valid" stat, but the conversion happens in rasterStatsToMetrics, so rasterStats is returning an object that looks like this: `[{histogram: {'1': 2, '2': 1, . . . }]` and is the whole histogram. I thought it would be strange if valid took that form, but because everything is based on the `categorical` bool, I could return the histogram as 'valid', but would have to update the typing of the 'valid' stat to support it.

Reply:
Okay, I like the idea then of continuing with trying out more robust stat types.  Did you or would you add histogram to the stat base typings?

Reply:
You could add it to `EXTRA_RASTER_STATS` in geoblaze.ts?